projects/ARM/devices/ARMv7/options

```diff
@@ -0,0 +1,21 @@
+################################################################################
+# setup system defaults
+################################################################################
+
+  # The TARGET_CPU variable controls which processor should be targeted for
+  # generated code.
+    TARGET_CPU="cortex-a8"
```

Comment:
I don't think we should be targeting a cpu

Reply:
I fully agree.

It's a bit tricky though ATM as config/arch.arm is built around the CPU (and defaults to cortex-a53 if not set) but at the end it only affects `-march` and ends up in `-mtune`.

I suggested on Slack to tackle that in a second step after we decided what we want - eg no `-mtune` at all, some typical, specific CPU for the arch (as it's now) or maybe `-mtune=generic-ARCH`